RELEASE.md

```diff
@@ -7,15 +7,17 @@ Please follow the established format:
 -->
 # Next release
 
+# Release 9.1.0
+
 ## Major features and improvements
+- Include expandAllPipelines in initial state. (#1896)
+- Allow Kedro-Viz commands to run from any sub directory within Kedro project. (#1871)
 
 ## Bug fixes and other changes
 
 - Upgrade the gitpod workspace-full to a newer version which includes both Node 18 and Python 3.11.5. (#1862)
```

Comment:
Since this is not user-facing change, can we keep it right at the end of the list of changes. Also, make it a bit concise? 
```suggestion
- Upgrade Gitpod to include Node 18 and Python 3.11. (#1862)
```